### ISSUE TITLE
Training on multiple GPUs?

### ISSUE THREAD

**Issue by sdy12318** [dmburd/S-DCNet#7]
Can I train this model with multiple GPUs?

**Comment by dmburd** [comment]
Yes, you can do that by wrapping the model into `nn.DataParallel`. Check out the [parallelism tutorial](https://pytorch.org/tutorials/beginner/former_torchies/parallelism_tutorial.html).
(Currently the code in this repo does not use `nn.DataParallel`.)
